include/beman/iterator_interface26/config.hpp.in

```diff
@@ -0,0 +1,6 @@
+#ifndef BEMAN_ITERATOR_INTERFACE26_CONFIG_HPP
```

Comment:
Would you like to remove 26 from codebase or from this PR (diff)?

Reply:
I intend to leave 26 in PR (and remove it separately). What I *do* want to remove in this PR is all references to *optional*.